### PR TITLE
DRILL-8248: Fix http_request for several rows

### PR DESCRIPTION
**Pull request by vdiravka**
# [DRILL-8248](https://issues.apache.org/jira/browse/DRILL-8248): Fix http_request for several rows

## Description

Fixes the limiting of the `http_request` to 1 row. The main issue is in creating `jsonLoader` for every row.

## Documentation
NA

## Testing
Unit test added. And tested manually with Drill Remote debug tool.

### CODE REVIEW

File: exec/java-exec/src/main/java/org/apache/drill/exec/store/easy/json/loader/SingleElementIterator.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package org.apache.drill.exec.store.easy.json.loader;
```
Comment (cgivre) [inline exec/java-exec/src/main/java/org/apache/drill/exec/store/easy/json/loader/SingleElementIterator.java:1]: Need a license header.

**Review by cgivre — CHANGES_REQUESTED** [review]
@vdiravka Looks good.  I saw a missing license header.  Other than that, looks good pending CI.

**Review by cgivre — APPROVED** [review]
LGTM +1 (Pending CI)

**Review by vvysotskyi — CHANGES_REQUESTED** [review]
Could you please add a unit test for it?

**Review by vdiravka — COMMENTED** [review]
@vvysotskyi 
Unit test is added.

**Review by vvysotskyi — APPROVED** [review]
+1

**Pull request merged**